swupd/manifest.go

```diff
@@ -236,6 +236,12 @@ func ParseManifestFile(path string) (*Manifest, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Only iterative manifests have names with .I.
+	if strings.Contains(m.Name, ".I.") {
+		m.Type = ManifestIterative
+	}
+
```

Comment:
Unrelated to this patch, but we should probably set the types for everything that we parse. It seems like a bug that fact that this is left unset.